newIDE/app/src/MainFrame/Editors/StartPage/index.js

```diff
@@ -160,7 +238,8 @@ class StartPage extends BaseEditor {
                     }
                   />
                 </Line>
-                <Line alignItems="center">
+                {/*TODO Le className disparait au rendu*/}
```

Comment:
See my other comment, but just so you understand:

`<div className="blabla">, <span className="blabla">, <a className="blabla">, etc...` will work because they are DOM elements.
`<MyComponent className="blabal" />` is a *React component*. className is just a prop like any other: if you don't use a prop (by passing it down to children), then it's useless. React won't magically add it because your component might not even be composed of any DOM element - in a lot of cases, React components are just rendering... other React components :)